### PR TITLE
GitHub authentication

### PR DESCRIPTION
**Pull request by wakwak3125**
## WHAT
- Implement GitHub auhentication

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -4,3 +4,5 @@ APP_BASIC_AUTH_USERNAME=admin
 APP_BASIC_AUTH_PASSWORD=password
 APP_DATABASE_URL=postgres://torinos_db/maindb?sslmode=disable
 APP_HONEYBADGER_API_KEY=
+APP_GITHUB_CLIENT_ID=
```
Comment (creasty) [inline .env.sample:7]: Could you also update here accordingly?
https://github.com/torinos-io/api/blob/fb46716b20c143750e51d43ceb5320efbd1ef3b6/kubernetes/deployment.yml#L56-L60

File: server/route/oauth.go [hunk 2]
```diff
@@ -4,18 +4,48 @@ import (
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+
+	"github.com/go-errors/errors"
```
Comment (creasty) [inline server/route/oauth.go]: Misordered...

**Comment by wakwak3125** [comment]
@creasty 
Thank you for reviewing!
I fixed your findings.

**Review by creasty — APPROVED** [review]
👍

**Pull request merged**